Easydict/objc/Service/Model/EZConstKey.h

```diff
@@ -44,6 +44,15 @@ static NSString *const EZCustomOpenAIValidModelsKey = @"EZCustomOpenAIValidModel
 // // Built-in AI
 static NSString *const EZBuiltInAIModelKey = @"EZBuiltInAIModelKey";
 
+// Gemini
+static NSString *const EZGeminiAPIKey = @"EZGeminiAPIKey";
+static NSString *const EZGeminiTranslationKey = @"EZGeminiTranslationKey";
+static NSString *const EZGeminiDictionaryKey = @"EZGeminiDictionaryKey";
+static NSString *const EZGeminiSentenceKey = @"EZGeminiSentenceKey";
+static NSString *const EZGeminiServiceUsageStatusKey = @"EZGeminiServiceUsageStatusKey";
+static NSString *const EZGeminiModelKey = @"EZGeminiModelKey";
+static NSString *const EZGeminiAvailableModelsKey = @"EZGeminiAvailableModelsKey";
+static NSString *const EZGeminiValidModelsKey = @"EZGeminiValidModelsKey";
```

Comment:
We no longer need to add this redundant template constant. We should use dynamic variables instead. Further optimization can be done here later, but for now, we can write it like this, for example, `validModelsStoredKey(.gemini)`.

```swift
func validModelsStoredKey(_ serviceType: ServiceType) -> String {
    storedKey("ValidModels", serviceType: serviceType)
}
```